solutions/cpython-tests/test_config_v3.8.11/patch

```diff
@@ -84,6 +84,18 @@ index effac97..7d8cb89 100644
      def test_lock_conflict(self):
          # Fork off a child process that will lock the mailbox temporarily,
          # unlock it and exit.
+diff --git a/Lib/test/test_poll.py b/Lib/test/test_poll.py
+index ef966bf..34a227b 100644
+--- a/Lib/test/test_poll.py
++++ b/Lib/test/test_poll.py
+@@ -117,6 +117,7 @@ class PollTests(unittest.TestCase):
+     # Another test case for poll().  This is copied from the test case for
+     # select(), modified to use poll() instead.
+ 
++    @unittest.skip("Temporarily disabled in Mystikos")
```

Comment:
Could you add the reason for disabling - unsupported mmap from child before execve